Wikipedia/Code/ImageDimmingExampleViewController.swift

```diff
@@ -20,6 +20,6 @@ extension ImageDimmingExampleViewController: Themeable {
         }
         
         view.backgroundColor = theme.colors.baseBackground
-        view.alpha = theme.imageOpacity
+        view.subviews.first?.alpha = theme.imageOpacity
```

Comment:
Good catch & fix - probably should use an IBOutlet for the image view in case the view is modified in the future and subviews.first is no longer the image view